tests/cpp_api/basic_test/basic_test.cpp

```diff
@@ -160,8 +160,11 @@ int main(int argc, const char **argv) {
         int counter = 0;
 
         while ((test_case == 0) ? !rocalIsEmpty(handle) : (counter < run_len[test_id])) {
-            if (rocalRun(handle) != 0)
+            auto status = rocalRun(handle);
+            if (status != 0) {
```

Comment:
please remove double if and check only the second one

Reply:
Done